modules/kernel/src/main/java/org/opencastproject/kernel/mail/SmtpService.java

```diff
@@ -166,10 +166,32 @@ private void sendTestMessage(String recipient) throws MessagingException {
    *           if sending the message failed
    */
   public void send(String to, String subject, String body) throws MessagingException {
+    send(to, subject, body, false);
+  }
+
+  /**
+   * Method to send a message
+   *
+   * @param to
+   *          Recipient of the message
+   * @param subject
+   *          Subject of the message
+   * @param body
+   *          Body of the message
+   * @param isHTML
+   *          Is the body of the message in HTML
+   * @throws MessagingException
+   *           if sending the message failed
+   */
+  public void send(String to, String subject, String body, Boolean isHTML) throws MessagingException {
     MimeMessage message = createMessage();
     message.addRecipient(RecipientType.TO, new InternetAddress(to));
     message.setSubject(subject);
-    message.setText(body);
+    if (isHTML) {
+        message.setContent(body, "text/html");
```

Comment:
I would make this a constant, otherwise looks good!

Reply:
:) Yes that's better.